runtime/gc_modron_startup/mmparseXgc.cpp

```diff
@@ -1123,6 +1123,15 @@ j9gc_initialize_parse_gc_colon(J9JavaVM *javaVM, char **scan_start)
 		goto _exit;
 	}
 		
+	if(try_scan(scan_start, "dynamicBreadthFirstScanOrdering")) {
+		extensions->scavengerScanOrdering = MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST;
+		/* Below options are required options for dynamicBreadthFirstScanOrdering */
+		extensions->scavengerAlignHotFields = false;
+		extensions->configurationOptions._gcPolicy = gc_policy_gencon;
```

Comment:
if dynamicBreadthFirstScanOrdering is provided along with say Balanced GC policy, the latter should take precedence, so that dynamicBreadthFirstScanOrdering  is ignored